### DISCUSSION TITLE
Clangd suppress findings

### DISCUSSION THREAD

**Discussion by ngunderson** [clangd/clangd#1122] [Q&A] [answered]
Currently I am using clangd 13.0 on a project with a cross-platform embedded toolchain. Every now and then I get some errors because clangd isn't quite the same as my embedded toolchain. One such error is this.
`In included file: token is not a valid binary operator in a preprocessor subexpression clang(pp_expr_bad_token_binop)`

Is there anyways to suppress some errors I find as nuisance errors? I have tried making my clangd configuration like below with no success.
```
Diagnostics:
    Suppress: ['pp_expr_bad_token_binop']
```

**Comment by HighCommander4** [comment, marked as answer]
The syntax for suppressing the error looks fine, however clangd <= 13 has a bug (#906) where the suppression mechanism fails for errors that occur in included header files -- perhaps that's the case for you?

That bug is fixed in clangd 14.

**Reply by ngunderson** [reply]
Upgrading to clangd 14, I did not have any issues. Thank you!
